iet.so/src/action.cpp

```diff
@@ -375,20 +375,22 @@ bool iet_action::get_all_common_config_keys(void) {
  * 
  */
 
-void iet_action::hip_to_smi_indices(){
+void iet_action::hip_to_smi_indices(void) {
```

Comment:
C++ doesnt need void in function parameter. Empty parameter list means funtion expects nothing. In C, this makes sense as empty means more than 0 args

Reply:
True. Just being explicit here :-)